configure.ac

```diff
@@ -304,6 +304,7 @@ case $host in
      fi
 
      CPPFLAGS="$CPPFLAGS -D_MT -DWIN32 -D_WINDOWS -DBOOST_THREAD_USE_LIB"
+     LIBS="$LIBS -lntdll"
```

Comment:
I believe it would be cleaner to be consistent with lines 270-290. Could we use `AC_CHECK_LIB` here too, for `ntdll`?

Reply:
Done, please double check if we are aligned on this.